CMakeLists.txt

```diff
@@ -81,6 +81,12 @@ if(APPLE OR WIN32)
     set(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
 endif()
 
+if (UNIX)
```

Comment:
How does this work?  Does it remove it when cmake configuration is rerun?  How often is this going to be removed.  Also, the jira doesn't really explain why you would do this.